polygon-digital-carbon/src/KlimaAggregator.ts

```diff
@@ -1,7 +1,7 @@
 import { MossRetired } from '../generated/RetireMossCarbon/RetireMossCarbon'
 import { ToucanRetired } from '../generated/RetireToucanCarbon/RetireToucanCarbon'
 import { C3Retired } from '../generated/RetireC3Carbon/RetireC3Carbon'
-import { CarbonRetired } from '../generated/KlimaInfinity/KlimaInfinity'
+import { CarbonRetired, CarbonRetired1 as CarbonRetiredTokenId } from '../generated/KlimaInfinity/KlimaInfinity'
```

Comment:
What's with `CarbonRetired1`? I don't see that function defined in the ABI for KlimaInfinity...

Reply:
This is the new event emission that shows up on the Aggregator due to emitting a token ID along with the address for a carbon project for ICR retirements. https://github.com/KlimaDAO/klimadao-solidity/blob/main/src/infinity/libraries/Bridges/LibICRCarbon.sol#L15